lib/openscap_parser/value_definitions.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/value_definition'
+require 'active_support/concern'
+
+module OpenscapParser
+  # Methods related to parsing values
+  module ValueDefinitions
+    extend ActiveSupport::Concern
+
+    included do
+      def value_definition_objects
```

Comment:
Why does this needs to be evaluated upon inclusion?

Reply:
Nevermind, I see...copy-paste. Please go with the original (bad) pattern then :shrug: